cli/flags.go

```diff
@@ -108,6 +110,7 @@ func configureFlags(root *cobra.Command) error {
 		"latest":         "TCTEST_LATESTBUILD",
 		"skip-queue":     "TCTEST_SKIP_QUEUE",
 		"open":           "TCTEST_OPEN_BROWSER",
+		"label":          "",
```

Comment:
we should have an env for this
```suggestion
		"label":          "TCTEST_BUILD_LABELS",
```